lib/tasks/email_and_unsubscribe_archived_topics.rake

```diff
@@ -0,0 +1,223 @@
+namespace :archived_topics do
+  desc "Send a bulk email to subscribers to these archived Specialist Topics."
+  task :send_emails, [:check_output] => :environment do |_t, args|
+    args.with_defaults(check_output: "false")
+    topic_urls.each_with_index do |topic, index|
+      sub_list = SubscriberList.find_by(url: topic[:url])
+      if !sub_list
+        puts "No SubscriberList for #{topic[:url]}"
+      else
+        puts "Sending email for #{sub_list.url} (ID: #{sub_list.id})"
+        puts "#{sub_list.subscribers.count} in this list"
+
+        subject = "Update from GOV.UK for: #{sub_list.title}"
+
+        body = <<~BODY
+          Update from GOV.UK for:
+
+          #{sub_list.title}
+
+          _________________________________________________________________
+
+          You asked GOV.UK to email you when we add or update a page about:
+
+
+          #{sub_list.title}
+
+          This topic has been archived. You will not get any more emails about it.
+
+          You can find more information about this topic at [#{topic[:redirect_title]}](#{topic[:redirect]}).
+        BODY
+
+        if index.zero? && args[:check_output] == "check"
+          puts "==============="
+          puts "CHECK OUTPUT DRY RUN"
+          puts "First SubscriberList..."
+          puts "The subscribers will see:"
+          puts "subject: #{subject}"
+          puts "body: #{body}"
+          puts "First recipient #{sub_list.subscribers.first.address}" if sub_list.subscribers.count.positive?
+          puts "==============="
+        else
+          email_ids = BulkSubscriberListEmailBuilder.call(
+            subject: subject,
+            body: body,
+            subscriber_lists: [sub_list],
+          )
+
+          email_ids.each do |id|
+            SendEmailWorker.perform_async_in_queue(id, queue: :send_email_immediate)
+          end
+
+          puts "Destroying subscription list #{sub_list.url} (ID: #{sub_list.id})"
```

Comment:
Because I'm a bit scared of deleting things (and email alert api in general I think!) I would probably have done the destruction of the subscription lists as a clean up task after everyone has been emailed. So this task would be to send the email, and unsubscribe users to the old lists ([this](https://github.com/alphagov/email-alert-api/pull/1681/files) sort of thing). And then a second task would be to delete them.

Reply:
That was originally the plan but felt that it was being too delayed and also once process would eventually how we want to do it ultimately in https://trello.com/c/CQlLYnrF/296-automate-unsubscribing-from-archived-specialist-topics